src/SDKs/CognitiveServices/CognitiveServices.Tests/CognitiveServices.Tests.csproj

```diff
@@ -27,4 +27,13 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
@felixwa in order to use ResourceManagement, you will have to make that change inside CognitiveServicesManagementTestUtilities.cs, also please remove Compiler Constant #DNX451 from your test/SDK project.
The only constant you should be using is
#if FullNetFx (for Desktop framework version e.g. .NET 4.5.2 .NET 4.6 etc) and if you are targeting .NET Standard, please use
#if !FullNetFx